terraform/shared/shared.tf.json.template.py

```diff
@@ -16,7 +16,7 @@
         'aws_s3_bucket': {
             'shared_cloudtrail': {
                 **provider_fragment(config.cloudtrail_s3_bucket_region),
-                'bucket': f'edu-ucsc-gi-{aws.account_name}-cloudtrail',
+                'bucket': config.qualified_bucket_name(aws.account_name, 'cloudtrail'),
```

Comment:
Find a way so that you don't have to pass the account name. The method doesn't *have* to reside in the `Config` class.